config/config.ts

```diff
@@ -0,0 +1,15 @@
+import * as fs from 'fs'
+
+let configFile
+let localConfig
+
+// If project is being run locally, the developer will have their API key in the json file
+try {
+  configFile = fs.readFileSync('./config/hollowbot.config.json', 'utf8')
+  localConfig = JSON.parse(configFile)
+} catch (error) {
+  // Project doesn't have a local config, we'll look for an environmental variable set via the Deployment medium
+  console.log('Local config is not available! -- Relying on environmental variables on the container.')
```

Comment:
> on the container

HollowBot doesn't have to run in a container always, right? We can probably just say that it is relying on environment variables, regardless of where they're coming from.